.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on pull requests to a set of branches.
+on:
+  pull_request:
+    branches: [
+      # This is our working branch
+      develop,
+      # This is where pull requests from "bors r+" are built.
+      staging,
+      # This is where pull requests from "bors try" are built. 
+      trying,
+    ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # no freebsd support for now =\
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: set up go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.13.12'
+    - name: test build
+      run: make
+    # - name: unit tests (except app test)
+    #   run: make test-no-app-test
+    # - name: app test
+    #   run: make test-only-app-test
+    - name: fmt, tidy, lint
+      # lint gets confused by windows line endings so skip it on windows
+      if: matrix.os != 'windows-latest'
```

Comment:
wouldn't it be better to add a windows version or fix the lint to work on both ? it is confusing as until I read this line I was sure it runs perfectly on windows as well..

Reply:
Linting by definition should not be platform-specific though, right? It seems sufficient to run a lint check on a single platform